### PR TITLE
Add krest parachain (kusama) 

### PR DESCRIPTION
**Pull request by kamrangul2404**
This PR adds new parachain for kusama

### CODE REVIEW

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 1]
```diff
@@ -835,6 +835,16 @@ export const prodParasKusamaCommon: EndpointOption[] = [
     ui: {
       logo: nodesBridgeHubBlackSVG
     }
+  },
+  {
+    homepage: 'https://krest.peaq.network/',
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:840]: This section is for common-good & system parahains.

You need to add it above.

Comment (kamrangul2404) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: Thankyou done the changes , kindly review now

**Review by sfffaaa — APPROVED** [review]

File: packages/apps-config/src/endpoints/productionRelayKusama.ts (hunk 1 above)
Comment (kamrangul2404) [reply packages/apps-config/src/endpoints/productionRelayKusama.ts]: @jacogr kindly let us know if there is anything pending from our-side

File: packages/apps-config/src/endpoints/productionRelayKusama.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ import type { EndpointOption } from './types';
...
+import { nodesApronPNG, nodesAresMarsPNG, nodesBajunPNG, nodesBasiliskPNG, nodesBifrostSVG, nodesBitcountryPNG, nodesBridgeHubBlackSVG, nodesCalamariPNG, nodesCrabSVG, nodesDatahighwayPNG, nodesEncointerBlueSVG, nodesImbuePNG, nodesIntegriteeSVG, nodesIpciSVG, nodesKabochaSVG, nodesKhalaSVG, nodesKrestPNG, nodesLitmusPNG, nodesLoomNetworkPNG, nodesLuhnPNG, nodesMoonriverSVG, nodesParallelSVG, nodesPicassoPNG, nodesPichiuPNG, nodesPolkasmithSVG, nodesQuartzPNG, nodesRobonomicsSVG, nodesSakuraSVG, nodesShadowSVG, nodesSnowPNG, nodesSoraSubstrateSVG, nodesStatemineSVG, nodesSubgameSVG, nodesSubsocialXSVG, nodesTrustbasePNG, nodesZeitgeistPNG, nodesZeroSVG } from '../ui/logos/nodes';
```
Comment (jacogr) [inline packages/apps-config/src/endpoints/productionRelayKusama.ts:8]: This file does not seem to have been added?

https://github.com/polkadot-js/apps/actions/runs/4285116528/jobs/7483319279#step:4:527

(Or rather [yarn build:images](https://github.com/polkadot-js/apps/tree/master/packages/apps-config#endpoints) was not run locally and the file checked-in?)

**Comment by jacogr** [comment]
This seems to be a 1MB image? https://github.com/polkadot-js/apps/actions/runs/4305591526/jobs/7519801674#step:4:516

(It is obviously slightly problematic since it gets served to all clients. A resize would be good to make the build pass without errors)

**Comment by lavish0000** [comment]
Hey @jacogr , Thanks again for your help. Please let me know if need anything else from our side.

**Review by jacogr — APPROVED** [review]

**Pull request merged**